### PR TITLE
Migration Guide Edits

### PR DESCRIPTION
**Pull request by ryanseams**
Moving information consistent across migrations to overview section, and cleaning up the provider specific guides from initial import from Notion (formatting, links, navigation, etc.)

### CODE REVIEW

File: pages/docs/other-bits/tutorials/migration-guides/migrating-to-mixpanel-from-amplitude.md [hunk 1]
```diff
@@ -1,226 +1,8 @@
 # Migrating to Mixpanel from Amplitude
 
-If you’re reading this, congrats on considering making the switch from Amplitude to Mixpanel 🎊  This guide is intended to outline how the migration works, what to expect, and how to mitigate the risks and switching costs of a migration whilst accelerate time to value.
+If you haven't already, we recommend starting with our [Migration Guides Overview](https://docs.mixpanel.com/docs/other-bits/tutorials/migration-guides) as it details the key components of migrating to Mixpanel from other analytics tools. Below we outline specific steps and considerations when migrating from Amplitude.
```
Comment (mavlee) [inline pages/docs/other-bits/tutorials/migration-guides/migrating-to-mixpanel-from-amplitude.md]: here and elsewhere:

use a relative path (get rid of docs.mixpanel.com prefix)

File: pages/docs/other-bits/tutorials/migration-guides.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+### What a migration will solve
+
+- Cut-over sending live data to Mixpanel
+- Historical backfill of existing data (subject to what data can be modeled as events)
+- Integrates easily with existing event data collection methods - SDKs, CDPs, DWH, RETL
+
+### What a migration won’t solve
+
+- Existing issues with data trust/quality or data governance → As part of the migration, we recommend a data audit as the first step to only transfer your valuable data and clean up trust/quality issues
+- Reporting, Dashboards, and Saved Entities → As part of the migration, you should review your top used reports and dashboards from other tools and sit down with each team to re-build them in Mixpanel
```
Comment (PeishanFromMixpanel) [inline pages/docs/other-bits/tutorials/migration-guides.md:12]: shouldn't we say that our CS team can help?

File: pages/docs/other-bits/tutorials/migration-guides.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+### Technical migration of data
+
+#### Data audit
+
+We’ve found from experience that <20% of the data in a product analytics tool is used for 80%+ of the queries. This is especially true the longer you have been using a tool - over time teams add more and more tracking for new events and properties, and without strong data governance practices, you will inevitably have some messy data in your current analytics tool(s).
+
+In the spirit of making sense of the mess, it is not recommended that you bring all historical data into Mixpanel. A common practice is to leverage your current providers tooling to understand which reports, events, and properties are queried by your users. No queries in the past 30 days? These events and properties have probably gone stale - there is low value and high effort in brining them to Mixpanel, so cut them from your import and do not migrate the existing tracking.
```
Comment (PeishanFromMixpanel) [inline pages/docs/other-bits/tutorials/migration-guides.md]: * we don't recommend

File: pages/docs/other-bits/tutorials/migration-guides.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+After you’ve gotten rid of the obvious (the reports, events, and properties no one uses), you can fine tune this approach by doing user interviews with your top users/champions. These users can help you explicitly define the data they need brought along to Mixpanel (mapped to their key questions and KPIs) so you can focus on what matters. Because these users are the ones building reporting others use, capturing their use cases and making them change agents can be highly beneficial to your migration.
+
+This data audit step is optional, but highly recommended - It is a larger upfront investment to avoid higher maintenance costs in the future.
+
+#### Loading historical data
+
+We recommend loading a year’s worth (or less) of historical data during your migration. This will allow your team to review year-over-year trends easily and do historical analysis as needed.
+
+To backfill data, we recommend:
```
Comment (PeishanFromMixpanel) [inline pages/docs/other-bits/tutorials/migration-guides.md]: Should we reverse the order - bring in live data first, before we bring in historical data? that way we can properly ensure the right cutover without duplicates. and re historical data, we also recommend bring in just a small subset into a test project first, to ensure that we're setting the right id management that is replicable to Mixpanel's?

File: pages/docs/other-bits/tutorials/migration-guides.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+#### Migrating live data tracking
+
+Depending on your current setup, the steps for migrating your live data tracking will differ. Please see the appropriate provider specific guides for more details on how to migrate your existing live data tracking and get started with Mixpanel.
+
+- [Amplitude Migration Guide](https://docs.mixpanel.com/docs/other-bits/tutorials/migration-guides/migrating-to-mixpanel-from-amplitude)
+- [Google Analytics Migration Guide](https://docs.mixpanel.com/docs/other-bits/tutorials/migration-guides/migrating-to-mixpanel-from-google-analytics)
+
+### Change management migration of end users
+
+Once data is live, we shift our focus to change management and migrating the existing users. We mitigate risk here by:
```
Comment (PeishanFromMixpanel) [inline pages/docs/other-bits/tutorials/migration-guides.md]: I think maybe we should make it clear right at the top: for enterprise customers, our customer success team will work with you to do the migration and change management. - cos in this section we're talking about how CS will help, but we won't actually help everyone.

**Review by PeishanFromMixpanel — COMMENTED** [review]
oh @ryanseams could you edit the tutorials landing page too, to add the link to migration-guides?

File: pages/docs/other-bits/tutorials/migration-guides.md (hunk 4 above)
Comment (ryanseams) [reply pages/docs/other-bits/tutorials/migration-guides.md]: sure, moving this instead to be live, audit, historical

File: pages/docs/other-bits/tutorials/migration-guides.md (hunk 5 above)
Comment (ryanseams) [reply pages/docs/other-bits/tutorials/migration-guides.md]: these recommendations should apply whether a customer has customer success or not. i rewrote the intro to make it clear - these same steps to mitigate risk can be done via self serve

File: pages/docs/other-bits/tutorials/migration-guides.md (hunk 2 above)
Comment (ryanseams) [reply pages/docs/other-bits/tutorials/migration-guides.md:12]: it does later. but this doc is not meant as an advertisement for cs and what we can do to help. its the best practices so anyone can learn and self-serve

**Pull request merged**